PublicMapping/districtbuilder
# Theme-UI componentization pass

## Overview

Refactors some of the existing UI into shared components and reworks the markup of every screen to use `theme-ui`

### Demo

#### Registration page

![localhost_3003_projects_5c102dbc-4a92-47fd-a50a-fb8199cf7d24](https://user-images.githubusercontent.com/4432106/82045662-77033a00-967d-11ea-9d4c-0bdbace3ea0a.png)
![localhost_3003_projects_5c102dbc-4a92-47fd-a50a-fb8199cf7d24 (3)](https://user-images.githubusercontent.com/4432106/82045681-81253880-967d-11ea-9921-d45cc4857846.png)

#### Create project page

![localhost_3003_ (10)](https://user-images.githubusercontent.com/4432106/82046309-a36b8600-967e-11ea-85ca-eee04125b258.png)
![localhost_3003_ (11)](https://user-images.githubusercontent.com/4432106/82046306-a2d2ef80-967e-11ea-9cad-3de06292ee9c.png)
![localhost_3003_ (8)](https://user-images.githubusercontent.com/4432106/82046312-a36b8600-967e-11ea-82c7-e99f85a8d0a9.png)
![localhost_3003_ (7)](https://user-images.githubusercontent.com/4432106/82046313-a4041c80-967e-11ea-9f55-93dea66459d8.png)
![localhost_3003_ (9)](https://user-images.githubusercontent.com/4432106/82046310-a36b8600-967e-11ea-9d63-d0fab32cba24.png)

#### Home page

![localhost_3003_ (12)](https://user-images.githubusercontent.com/4432106/82046499-04935980-967f-11ea-96fb-6e8eae5fb5ef.png)


### Notes

 - Not sure I necessarily found the right set of reusable components with the `Field` component(s) - up for other ideas.
 - Purposefully kept a light touch on the Project page, which is under development right now (#83) and the Home page, which wasn't built out enough to style more fully.

## Testing Instructions

- `scripts/update`
- Visit every page. Do all the things. They should work as before, but now look nicer.

Closes #99
